src/main.rs

```diff
@@ -261,7 +261,8 @@ fn create_app<'a, 'b>(
                 .long("seccomp")
                 .takes_value(true)
                 .possible_values(&["true", "false"])
-                .default_value("true"),
+                .default_value("true")
+                .group("vm-config"),
```

Comment:
seccomp filters are dealt with at the VMM level so although we need to propagate the value into the vm and device manager it shouldn't be configurable per-vm.

Reply:
Agreed.